mlflow/keras.py

```diff
@@ -421,6 +421,12 @@ def on_epoch_end(self, epoch, logs=None):
         def on_train_end(self, logs=None):
             try_mlflow_log(log_model, self.model, artifact_path='model')
 
+    def early_stop_check(callbacks):
+        for callback in callbacks:
+            if isinstance(callback, keras.callbacks.callbacks.Callback):
```

Comment:
This is probably an ignorant question, but is this definitely an early stopping callback? it seems to just check if it's a keras callback, which could be many things.

Reply:
Not ignorant - I copy pasted the wrong class (the parent) by accident. Thanks for the catch!